util/dbterror/terror_test.go

```diff
@@ -47,44 +41,44 @@ func (s *testkSuite) TestErrorRedact(c *C) {
 	QuestionMark := "?"
 
 	err := class.NewStd(errno.ErrDupEntry).GenWithStackByArgs(SensitiveData, NoSensitiveValue)
-	c.Assert(strings.Contains(err.Error(), genErrMsg(errno.MySQLErrName[errno.ErrDupEntry].Raw, QuestionMark, NoSensitiveValue)), IsTrue)
+	assert.True(t, strings.Contains(err.Error(), genErrMsg(errno.MySQLErrName[errno.ErrDupEntry].Raw, QuestionMark, NoSensitiveValue)))
```

Comment:
You can make use of `require.Contains` for a better assertion and fail report.

Reply:
okay, thanks a lot, already updated